Client/Frontend/Browser/Tab.swift

```diff
@@ -54,7 +55,7 @@ class Tab: NSObject {
     }
 
     var tabState: TabState {
-        return TabState(isPrivate: _isPrivate, desktopSite: desktopSite, isBookmarked: isBookmarked, url: url, title: displayTitle, favicon: displayFavicon)
+        return TabState(isPrivate: _isPrivate, desktopSite: desktopSite, isBookmarked: isBookmarked, isNoImageMode: isNoImageMode, url: url, title: displayTitle, favicon: displayFavicon)
```

Comment:
Much of the need goes away following https://github.com/mozilla/firefox-ios/pull/1816 . Rebase and have look how at `HomePageAccessors` are used.